saleor/graphql/product/mutations/attributes.py

```diff
@@ -30,6 +30,9 @@ class ProductAttributeAssignInput(graphene.InputObjectType):
     type = ProductAttributeType(
         required=True, description="The attribute type to be assigned as."
     )
+    variant_selection = graphene.Boolean(
+        required=False, description="Whether attribute is allowed in variant selection."
```

Comment:
I think it might be useful to include here the types that can be used in variant selection. We could generate that from the `ALLOWED_IN_VARIANT_SELECTION` list. Also we should add `ADDED_IN_31` note.

Reply:
Ofc 👍 . Done.